typescript/cli/src/config/chain.ts

```diff
@@ -69,6 +70,30 @@ export async function createChainConfig({
     default: name[0].toUpperCase() + name.slice(1),
   });
 
+  const technicalStack = (await select({
+    choices: Object.entries(ChainTechnicalStack).map(([_, value]) => ({
+      value,
+    })),
+    message: 'Select the chain technical stack',
+    pageSize: 10,
+  })) as ChainTechnicalStack;
+
+  const arbitrumNitroMetadata: Partial<ChainMetadata> = {};
```

Comment:
I wonder if we want to be more specific about the arb nitro metadata, maybe ChainMetadata['index'] as the type

also, i think we should ask for tech stack/index.from right at the end after asking if it's a testnet